10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
+Each library has a set of feature/index pairs associated with it, and these mappings are stored in the Library Features Table.
 
-The `Library Features` feature names contain the following pieces of information in an underscore separated fashion: 1) official gene symbol of the measured protein (or the protein instead if desired - otherwise other names will be stored as aliaes), 2) the unique HIMC id (e.g. HIMC-1), and the lot number (e.g. lot-1). 
+The `Feature` names contain the following underscore-delimited components:
+1) official gene symbol of the measured protein (or the protein instead if desired - otherwise other names will be stored as aliases)
+2) the unique HIMC id (e.g. HIMC-1), and the lot number (e.g. lot-1).
 
-The `Index` column contains the oligo index IDs that can be looked up in a different `Features Table`.
+The `Index` column contains the oligo index IDs that can be looked up in [the `Features Table`][Library Features Table].
 
 # 10x Required CSVs
 
-### Sample Sheet CSV
+## Sample Sheet CSV
 | Lane| Sample | Index |
 |---|---|---|
 | 1  | S1_GEX | SI-GA-A3 |
 
-This table is in the format of the simple sample sheet for mkfastq. `Lane` refers to the 10x chip lane. `Sample` refers to our loading sample name (it will be appended with the BCL name automatically (I think) by mkfastq). `Index` refers to the oligos used to de-multiplex the BCL - in order to process custom oligos (e.g. ADT/HTO) we will pass the actual oligo index sequences in the `Index` column. 
+This table is in the format of the "simple samplesheet" consumed by `cellranger mkfastq`:
+- `Lane` refers to the 10x chip lane.
+- `Sample` refers to our loading sample name (the BCL name will be appended automatically by `mkfastq`; TODO: verify this).
+- `Index` refers to the oligos used to de-multiplex the BCL
+  - When processing custom oligos (e.g. ADT/HTO), pass the actual oligo sequences here.
 
-### Libraries CSV
-|  fastqs | sample  |  library_type |
+## Libraries CSV
+|  FASTQs | Sample  |  Library Type |
 |---|---|---|
 | /path/to/fastqs/ | S1_GEX_BCL-1 | Gene Expression |
 | /path/to/fastqs/ | S1_HTO_BCL-1 | Custom |
 
-`fastqs` give a path to the demultiplexed FASTQ files (cannot have comma-delimiited paths) - more than one path requres an additional row. `sample` is the sampe name assigned in the mkfastq simple sample sheet. `library_type` is self explanatory.
+- `FASTQs`: path to demultiplexed FASTQ files (cannot have comma-delimiited paths; more than one path requres an additional row).
+- `Sample` is the sample name assigned in the `mkfastq` simple samplesheet.
+- `Library Type` should be self-explanatory. <!-- Can we enumerate the possibilities here? Are they referenced anywhere else, or is this just a free-form "Notes" type of field? -->
```

Comment:
Can we enumerate the possibilities here? Are they referenced anywhere else, or is this just a free-form "Notes" type of field?

Reply:
This can be found in the 10x docs (https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis). This is the description of the `Library Type` column from 10x: 

"
The FASTQ data will be interpreted using the rows from the feature reference file that have a ‘feature_type’ that matches this library_type. This field is case-sensitive, and must match a valid library type as described in the Library / Feature Types section. Must be Gene Expression for the gene expression libraries. Must be one of Custom, Antibody Capture, or CRISPR Guide Capture for Feature Barcoding libraries.
"